paymentsheet/src/main/java/com/stripe/android/customersheet/CustomerSheet.kt

```diff
@@ -30,23 +31,27 @@ import javax.inject.Inject
  * [CustomerAdapter].
  */
 @ExperimentalCustomerSheetApi
-class CustomerSheet @Inject internal constructor(
+class CustomerSheet internal constructor(
     private val application: Application,
     lifecycleOwner: LifecycleOwner,
     activityResultRegistryOwner: ActivityResultRegistryOwner,
+    viewModelStoreOwner: ViewModelStoreOwner,
```

Comment:
Should we pass in a saved state handle, and do the storage directly instead?

Reply:
It feels like that would be less confusing and easier to follow, but I might be missing something too.

Reply:
We could but `SavedStateHandle` can generally only be created within a view model context so we would have to create the view model for all three implementations (activity, fragment, and compose). On top of that, there are additional dependencies that I want to attach to this view model that shouldn't be re-created on every config change.